### ISSUE TITLE
Error: must specify version

### ISSUE THREAD

**Issue by wuxianliang** [SimulatedGREG/electron-vue#12]
Dear @SimulatedGREG 
  When I build with
```bash
npm run build:darwin
```
one error occurs.
```bash
Building electron app(s)...

Error: must specify version
    at download (/Users/wxl/projects/bg/node_modules/electron-download/index.js:16:27)
    at Array.<anonymous> (/Users/wxl/projects/bg/node_modules/electron-packager/index.js:121:7)
    at each (/Users/wxl/projects/bg/node_modules/run-series/index.js:17:24)
    at next (/Users/wxl/projects/bg/node_modules/rimraf/rimraf.js:74:7)
    at CB (/Users/wxl/projects/bg/node_modules/rimraf/rimraf.js:110:9)
    at /Users/wxl/projects/bg/node_modules/rimraf/rimraf.js:136:14
    at FSReqWrap.oncomplete (fs.js:123:15)
Build(s) successful!
undefined

DONE

```
I think there is a problem with config.js, because when I change 
```javascript
building:{
  'app-version': pkg.version,
  ......
}
```
to 
```javascript
building:{
  version: '1.3.2', // pkg.version,
  ......
}
```
which is the version of electron-prebuilt. It works!!!

**Comment by SimulatedGREG** [comment]
The default setup uses the version number from `./app/package.json`. Can you confirm that is a proper semver? Do you have this in a repo I could look into as well?

**Comment by wuxianliang** [comment]
When I only change "version" in package.json to "version": "1.1.1", it report error also.
When I add a new term version: '1.2.1'into building:{ }  in config.js, it download electron-v1.2.1 during the building process. So I think we have to specify the version of electron-prebuild.

**Comment by SimulatedGREG** [comment]
@wuxianliang

Found a pull request with `electron-packager` that sort of broke getting the electron version automatically (it would seem), which is live in `v7.5.0`. For now, you can continue to manually set the electron version (like you demonstrated above), or you can install the previous version of `electron-packager` by doing `npm i --save-dev electron-packager@7.4.0`. 

Here's the PR...
[https://github.com/electron-userland/electron-packager/pull/435#issuecomment-238036368](https://github.com/electron-userland/electron-packager/pull/435#issuecomment-238036368)

**Comment by SimulatedGREG** [comment]
@wuxianliang 

Did some further investigation and found what broke in the new update. (You discovered something kinda nasty for them in my opinion... 😁)

**Comment by SimulatedGREG** [comment]
@wuxianliang 

Can confirm a fix is being released rather soon in [`electron-packager@7.5.1`](https://github.com/electron-userland/electron-packager/pull/435#issuecomment-238038234). Closing this now.

**Comment by SimulatedGREG** [comment]
`electron-packager@7.5.1`unfortunately did not fix the issue. 

[https://github.com/electron-userland/electron-packager/pull/440#issuecomment-238043603](https://github.com/electron-userland/electron-packager/pull/440#issuecomment-238043603)

**Comment by SimulatedGREG** [comment]
Fixed with `electron-packager@7.6.0`.
